global-setup.js

```diff
@@ -0,0 +1,8 @@
+async function globalSetup() {
+  // SHOGun Application-ID e.g.:
+  process.env.ID = '42';
+  // Host of the Application including potential paths e.g.:
+  process.env.HOST = 'my.application.de';
+}
+
+export default globalSetup;
```

Comment:
Can we get this parameters from an `.env` file instead?

Reply:
Yes that is also already done in the follow-up MR that is already prepared. I could remove the whole file from this MR actually.